src/lib/util/buyLimit.ts

```diff
@@ -0,0 +1,68 @@
+import { Bank } from 'oldschooljs';
+import { Item } from 'oldschooljs/dist/meta/types';
+
+export function buyLimit({
+	buyLimitBank,
+	increaseFactor,
+	itemBeingBought,
+	quantityBeingBought,
+	baseCost,
+	absoluteLimit
+}: {
+	buyLimitBank: Bank;
+	/**
+	 * The price multiplier increases by 1 for each multiple of this amount they have bought. (1x, 2x, 3x, etc)
```

Comment:
This comment i made is not correct with the current algorithm,, it  should be something like,
'The price multiplier doubles for every multiple of this amount they have bought'